cmd/bootstrap/runner.go

```diff
@@ -545,6 +528,16 @@ func (r *runner) installOperator(ctx context.Context, helmClient helmclient.Inte
 
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("installing %#q", name))
 
+		// Set operator values so chart-operator DNS settings are correct.
+		// Merge with an empty set of values so YAML is parsed.
+		input := map[string][]byte{
+			"values": []byte(operatorValuesYAML),
+		}
+		values, err := helmclient.MergeValues(input, map[string][]byte{})
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
```

Comment:
Previously we just patched the chart-operator deployments to set the DNS settings but I hit problems with the status webhook.

So now we install the operators using control plane values and the chart templating takes care of the DNS settings.

Reply:
I guess this should be enough, as for `chart-museum` to get up, `app-operator` and `chart-operator` must be upas well, right? Does it make sense to explicitly wait on their deployments to report "ready" as well?

Reply:
Yes both operators need to be up and have processed the chartmuseum app CR. As we check the status of the app CR. So there is no benefit to using the wait flag when installing the operators with helmclient.

Maybe we should enable "helm install --wait" via app CRs. As that could be useful in some situations. But thats a bigger change and I think should be a separate issue.